### ISSUE TITLE
Wrong APP_PATH

### ISSUE THREAD

**Issue by zigomir** [vuejs/vitepress#15]
Following the steps from `README.md`

```sh
~/development/playground/vitepress_test
❯ echo '# Hello VitePress' > index.md

~/development/playground/vitepress_test
❯ npx vitepress
vitepress v0.2.0
vite v0.16.7
listening at http://localhost:3000

  Error: ENOENT: no such file or directory, stat '/Users/ziga/.npm/_npx/21240/lib/node_modules/vitepress/dist/client/app/index.html'
      at Object.statSync (fs.js:1035:3)
      at Object.statSync (/Users/ziga/.npm/_npx/21240/lib/node_modules/vitepress/node_modules/graceful-fs/polyfills.js:308:16)
      at Object.cachedRead (/Users/ziga/.npm/_npx/21240/lib/node_modules/vitepress/node_modules/vite/dist/utils/fsUtils.js:20:45)
      at /Users/ziga/.npm/_npx/21240/lib/node_modules/vitepress/dist/node/server.js:98:30
      at async /Users/ziga/.npm/_npx/21240/lib/node_modules/vitepress/node_modules/vite/dist/server/serverPluginHtml.js:51:9
      at async /Users/ziga/.npm/_npx/21240/lib/node_modules/vitepress/node_modules/vite/dist/server/serverPluginModuleRewrite.js:25:9
```

causes **Error: ENOENT**.

I believe it originates from [resolver's APP_PATH](https://github.com/vuejs/vitepress/blob/master/src/node/resolver.ts#L4)

**Comment by MDSLKTR** [comment]
`index.html` is not bundled. Might not be intended for dev mode because it might be served from memory, yet this is what is is trying to resolve.

**Comment by yyx990803** [comment]
Fixed in 0.3.1
